### ISSUE TITLE
Your message was not sent because "" is ignore you

### ISSUE THREAD

**Issue by XxPoNaGeXx** [kangarko/ChatControl-Pro#1353]
Your message was not sent because "" is ignore you

when trying to use /msg

I use bungee 

Here is my config https://paste.gg/p/anonymous/37ab0d9d7522402986de13a68da6cd5c

Config is exactly same on my servers

**Comment by XxPoNaGeXx** [comment]
Disabling ignore players in config then gives this error: 

```------------------------------------[ 15.08.2019 19:20:01 ]-----------------------------------
ChatControl 8.3.11 encountered an IllegalPluginAccessException
Running Paper 1.14.4-R0.1-SNAPSHOT and Java 1.8.0_201
Plugins: CoreProtect v2.17.5, LuckPerms v4.4.1, WorldEdit v7.0.1-SNAPSHOT;4540-edb2e91, PhantomSpawnControl v2.1.1, AdvancedBanAutoBan v1.0.1, BlockLocker v1.6.1, PlaceholderAPI v2.10.2, PlayerRespawnerPlugin v0.0.1-SNAPSHOT, NoChat v1.0, Spicord v1.0.4-SNAPSHOT, PlugMan v2.1.6, AdvancedBan v2.1.5-RELEASE, spark v1.3.0, BanAnnouncer v1.0.2, FarmLimiter v1.2, DiscordBotAPI v3.0, StaffFacilities v4.1.0, WorldGuard v7.0.1-SNAPSHOT;1945-b0e4cf1, AdvancedPortals v0.0.49, MVdWPlaceholderAPI v3.0.1, SuperTrails v9.7.1, DiscordChatWM v3.1.2, FeatherBoard v4.22.0, CraftingStore v2.2.0, LuckPermsMVdWHook v4.1-SNAPSHOT, Vault v1.7.2-b107, PlayerShopGUIPlus v1.3.1, TimeVote v1.42, ShopGUIPlus v1.19.6, MyWarp v3.0.2-SNAPSHOT+travis-b1784.git-f3ed0f39ba, Essentials v2.17.1.0, ArmorStandTools v3.4.0, SpigotLib v9.0.3, ProtocolLib v4.5.0-SNAPSHOT, BentoBox v1.5.3, EssentialsChat v2.17.1.0, StoneOres2 v0.1.1, EssentialsSpawn v2.17.1.0, PluginLibrary v1.2.1, HolographicDisplays v2.3.3-SNAPSHOT, CrateReloaded 2.0.14 by dinosaur, UltimateStacker v1.4.5, ChatControl v8.3.11
----------------------------------------------------------------------------------------------

More Information: 
Error reloading ChatControl 8.3.11
IllegalPluginAccessException Plugin attempted to register class org.bukkit.event.player.AsyncPlayerChatEvent while not enabled
	 at org.bukkit.plugin.SimplePluginManager.registerEvent(SimplePluginManager.java:608)
	 at org.mineacademy.chatcontrol.final.nul. .LPt5.const(SourceFile:69)
	 at org.mineacademy.chatcontrol.final.nul. .LpT4.const(SourceFile:57)
	 at org.mineacademy.chatcontrol.final.nul. .com4.const(SourceFile:832)
	 at org.mineacademy.chatcontrol.ChatControl.transient(SourceFile:275)
	 at org.mineacademy.chatcontrol.final.nul. .com4.static(SourceFile:744)
	 at org.mineacademy.chatcontrol.final.nul. .short.const(SourceFile:562)
	 at org.mineacademy.chatcontrol.final.nul. .LPt8.execute(SourceFile:305)
	 at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
	 at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:739)
----------------------------------------------------------------------------------------------


------------------------------------[ 15.08.2019 19:20:14 ]-----------------------------------
ChatControl 8.3.11 encountered an IllegalPluginAccessException
Running Paper 1.14.4-R0.1-SNAPSHOT and Java 1.8.0_201
Plugins: CoreProtect v2.17.5, LuckPerms v4.4.1, WorldEdit v7.0.1-SNAPSHOT;4540-edb2e91, PhantomSpawnControl v2.1.1, AdvancedBanAutoBan v1.0.1, BlockLocker v1.6.1, PlaceholderAPI v2.10.2, PlayerRespawnerPlugin v0.0.1-SNAPSHOT, NoChat v1.0, Spicord v1.0.4-SNAPSHOT, PlugMan v2.1.6, AdvancedBan v2.1.5-RELEASE, spark v1.3.0, BanAnnouncer v1.0.2, FarmLimiter v1.2, DiscordBotAPI v3.0, StaffFacilities v4.1.0, WorldGuard v7.0.1-SNAPSHOT;1945-b0e4cf1, AdvancedPortals v0.0.49, MVdWPlaceholderAPI v3.0.1, SuperTrails v9.7.1, DiscordChatWM v3.1.2, FeatherBoard v4.22.0, CraftingStore v2.2.0, LuckPermsMVdWHook v4.1-SNAPSHOT, Vault v1.7.2-b107, PlayerShopGUIPlus v1.3.1, TimeVote v1.42, ShopGUIPlus v1.19.6, MyWarp v3.0.2-SNAPSHOT+travis-b1784.git-f3ed0f39ba, Essentials v2.17.1.0, ArmorStandTools v3.4.0, SpigotLib v9.0.3, ProtocolLib v4.5.0-SNAPSHOT, BentoBox v1.5.3, EssentialsChat v2.17.1.0, StoneOres2 v0.1.1, EssentialsSpawn v2.17.1.0, PluginLibrary v1.2.1, HolographicDisplays v2.3.3-SNAPSHOT, CrateReloaded 2.0.14 by dinosaur, UltimateStacker v1.4.5, ChatControl v8.3.11
----------------------------------------------------------------------------------------------

More Information: 
Error reloading ChatControl 8.3.11
IllegalPluginAccessException Plugin attempted to register class org.bukkit.event.player.AsyncPlayerChatEvent while not enabled
	 at org.bukkit.plugin.SimplePluginManager.registerEvent(SimplePluginManager.java:608)
	 at org.mineacademy.chatcontrol.final.nul. .LPt5.const(SourceFile:69)
	 at org.mineacademy.chatcontrol.final.nul. .LpT4.const(SourceFile:57)
	 at org.mineacademy.chatcontrol.final.nul. .com4.const(SourceFile:832)
	 at org.mineacademy.chatcontrol.ChatControl.transient(SourceFile:275)
	 at org.mineacademy.chatcontrol.final.nul. .com4.static(SourceFile:744)
	 at org.mineacademy.chatcontrol.final.nul. .short.const(SourceFile:562)
	 at org.mineacademy.chatcontrol.final.nul. .LPt8.execute(SourceFile:305)
	 at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
	 at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:739)
----------------------------------------------------------------------------------------------


------------------------------------[ 15.08.2019 19:22:12 ]-----------------------------------
ChatControl 8.3.11 encountered an IllegalArgumentException
Running Paper 1.14.4-R0.1-SNAPSHOT and Java 1.8.0_201
Plugins: CoreProtect v2.17.5, LuckPerms v4.4.1, WorldEdit v7.0.1-SNAPSHOT;4540-edb2e91, PhantomSpawnControl v2.1.1, AdvancedBanAutoBan v1.0.1, BlockLocker v1.6.1, PlaceholderAPI v2.10.2, PlayerRespawnerPlugin v0.0.1-SNAPSHOT, NoChat v1.0, Spicord v1.0.4-SNAPSHOT, PlugMan v2.1.6, AdvancedBan v2.1.5-RELEASE, spark v1.3.0, BanAnnouncer v1.0.2, FarmLimiter v1.2, DiscordBotAPI v3.0, StaffFacilities v4.1.0, WorldGuard v7.0.1-SNAPSHOT;1945-b0e4cf1, AdvancedPortals v0.0.49, MVdWPlaceholderAPI v3.0.1, SuperTrails v9.7.1, DiscordChatWM v3.1.2, FeatherBoard v4.22.0, CraftingStore v2.2.0, LuckPermsMVdWHook v4.1-SNAPSHOT, Vault v1.7.2-b107, PlayerShopGUIPlus v1.3.1, TimeVote v1.42, ShopGUIPlus v1.19.6, MyWarp v3.0.2-SNAPSHOT+travis-b1784.git-f3ed0f39ba, Essentials v2.17.1.0, ArmorStandTools v3.4.0, SpigotLib v9.0.3, ProtocolLib v4.5.0-SNAPSHOT, BentoBox v1.5.3, EssentialsChat v2.17.1.0, StoneOres2 v0.1.1, EssentialsSpawn v2.17.1.0, PluginLibrary v1.2.1, HolographicDisplays v2.3.3-SNAPSHOT, CrateReloaded 2.0.14 by dinosaur, UltimateStacker v1.4.5, ChatControl v8.3.11
----------------------------------------------------------------------------------------------

More Information: 
Failed to execute command /msg kyleboyce_ test
IllegalArgumentException Plugin must be enabled to send messages
	 at org.bukkit.plugin.messaging.StandardMessenger.validatePluginMessage(StandardMessenger.java:538)
	 at org.bukkit.craftbukkit.v1_14_R1.entity.CraftPlayer.sendPluginMessage(CraftPlayer.java:1466)
	 at org.mineacademy.chatcontrol.final.nul. .lPT4.const(SourceFile:113)
	 at org.mineacademy.chatcontrol.final.nul. .lPT4.const(SourceFile:43)
	 at org.mineacademy.chatcontrol.final.nul. .AUX.const(SourceFile:165)
	 at org.mineacademy.chatcontrol.final.nul. .LPt8.execute(SourceFile:305)
	 at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
	 at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:739)
----------------------------------------------------------------------------------------------


------------------------------------[ 15.08.2019 19:23:04 ]-----------------------------------
ChatControl 8.3.11 encountered an IllegalArgumentException
Running Paper 1.14.4-R0.1-SNAPSHOT and Java 1.8.0_201
Plugins: CoreProtect v2.17.5, LuckPerms v4.4.1, WorldEdit v7.0.1-SNAPSHOT;4540-edb2e91, PhantomSpawnControl v2.1.1, AdvancedBanAutoBan v1.0.1, BlockLocker v1.6.1, PlaceholderAPI v2.10.2, PlayerRespawnerPlugin v0.0.1-SNAPSHOT, NoChat v1.0, Spicord v1.0.4-SNAPSHOT, PlugMan v2.1.6, AdvancedBan v2.1.5-RELEASE, spark v1.3.0, BanAnnouncer v1.0.2, FarmLimiter v1.2, DiscordBotAPI v3.0, StaffFacilities v4.1.0, WorldGuard v7.0.1-SNAPSHOT;1945-b0e4cf1, AdvancedPortals v0.0.49, MVdWPlaceholderAPI v3.0.1, SuperTrails v9.7.1, DiscordChatWM v3.1.2, FeatherBoard v4.22.0, CraftingStore v2.2.0, LuckPermsMVdWHook v4.1-SNAPSHOT, Vault v1.7.2-b107, PlayerShopGUIPlus v1.3.1, TimeVote v1.42, ShopGUIPlus v1.19.6, MyWarp v3.0.2-SNAPSHOT+travis-b1784.git-f3ed0f39ba, Essentials v2.17.1.0, ArmorStandTools v3.4.0, SpigotLib v9.0.3, ProtocolLib v4.5.0-SNAPSHOT, BentoBox v1.5.3, EssentialsChat v2.17.1.0, StoneOres2 v0.1.1, EssentialsSpawn v2.17.1.0, PluginLibrary v1.2.1, HolographicDisplays v2.3.3-SNAPSHOT, CrateReloaded 2.0.14 by dinosaur, UltimateStacker v1.4.5, ChatControl v8.3.11
----------------------------------------------------------------------------------------------

More Information: 
Failed to execute command /msg kyleboyce_ test
IllegalArgumentException Plugin must be enabled to send messages
	 at org.bukkit.plugin.messaging.StandardMessenger.validatePluginMessage(StandardMessenger.java:538)
	 at org.bukkit.craftbukkit.v1_14_R1.entity.CraftPlayer.sendPluginMessage(CraftPlayer.java:1466)
	 at org.mineacademy.chatcontrol.final.nul. .lPT4.const(SourceFile:113)
	 at org.mineacademy.chatcontrol.final.nul. .lPT4.const(SourceFile:43)
	 at org.mineacademy.chatcontrol.final.nul. .AUX.const(SourceFile:165)
	 at org.mineacademy.chatcontrol.final.nul. .LPt8.execute(SourceFile:305)
	 at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
	 at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:739)
----------------------------------------------------------------------------------------------


------------------------------------[ 15.08.2019 19:37:12 ]-----------------------------------
ChatControl 8.3.11 encountered an IllegalPluginAccessException
Running Paper 1.14.4-R0.1-SNAPSHOT and Java 1.8.0_201
Plugins: CoreProtect v2.17.5, LuckPerms v4.4.1, WorldEdit v7.0.1-SNAPSHOT;4540-edb2e91, PhantomSpawnControl v2.1.1, AdvancedBanAutoBan v1.0.1, BlockLocker v1.6.1, PlaceholderAPI v2.10.2, PlayerRespawnerPlugin v0.0.1-SNAPSHOT, NoChat v1.0, Spicord v1.0.4-SNAPSHOT, PlugMan v2.1.6, AdvancedBan v2.1.5-RELEASE, spark v1.3.0, BanAnnouncer v1.0.2, FarmLimiter v1.2, DiscordBotAPI v3.0, StaffFacilities v4.1.0, WorldGuard v7.0.1-SNAPSHOT;1945-b0e4cf1, AdvancedPortals v0.0.49, MVdWPlaceholderAPI v3.0.1, SuperTrails v9.7.1, DiscordChatWM v3.1.2, FeatherBoard v4.22.0, CraftingStore v2.2.0, LuckPermsMVdWHook v4.1-SNAPSHOT, Vault v1.7.2-b107, PlayerShopGUIPlus v1.3.1, TimeVote v1.42, ShopGUIPlus v1.19.6, MyWarp v3.0.2-SNAPSHOT+travis-b1784.git-f3ed0f39ba, Essentials v2.17.1.0, ArmorStandTools v3.4.0, SpigotLib v9.0.3, ProtocolLib v4.5.0-SNAPSHOT, BentoBox v1.5.3, EssentialsChat v2.17.1.0, StoneOres2 v0.1.1, EssentialsSpawn v2.17.1.0, PluginLibrary v1.2.1, ChatControl v8.3.11, HolographicDisplays v2.3.3-SNAPSHOT, CrateReloaded 2.0.14 by dinosaur, UltimateStacker v1.4.5
----------------------------------------------------------------------------------------------

More Information: 
Error reloading ChatControl 8.3.11
IllegalPluginAccessException Plugin attempted to register class org.bukkit.event.player.AsyncPlayerChatEvent while not enabled
	 at org.bukkit.plugin.SimplePluginManager.registerEvent(SimplePluginManager.java:608)
	 at org.mineacademy.chatcontrol.final.nul. .LPt5.const(SourceFile:69)
	 at org.mineacademy.chatcontrol.final.nul. .LpT4.const(SourceFile:57)
	 at org.mineacademy.chatcontrol.final.nul. .com4.const(SourceFile:832)
	 at org.mineacademy.chatcontrol.ChatControl.transient(SourceFile:275)
	 at org.mineacademy.chatcontrol.final.nul. .com4.static(SourceFile:744)
	 at org.mineacademy.chatcontrol.final.nul. .short.const(SourceFile:562)
	 at org.mineacademy.chatcontrol.final.nul. .LPt8.execute(SourceFile:305)
	 at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
	 at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:739)
----------------------------------------------------------------------------------------------


```

**Comment by kangarko** [comment]
First problem has been fixed, the other one is being dealt with in #1354
